ballgame.py

```diff
@@ -82,6 +85,7 @@ def __init__(self, pos, imagefile, hp):
         self.rect = self.image.get_rect()
         self.rect.center = pos
         self.v = [0,0]
+        self.speed = 5
```

Comment:
Why 5?  Shouldn't it be speed/2 or something?

Reply:
The balls go 10, so I put 5 for the chicken.  I'm OK with any speed you want.  When you say speed/2, do you mean half the ball speed?  We should probably refactor/rename our Sprite and BounceSprite classes soon.

Reply:
Just that the speed is passed in for chickens, do we want a default for the balls or to pass it in as well?

Reply:
Args for the chicken are pos, imagefile, hp at the moment.  But I was thinking of passing in speeds for target sprites